rlog.go

```diff
@@ -63,6 +63,18 @@ var levelNumbers = map[string]int{
 	"NONE":     levelNone,
 }
 
+type TraceLevel int
+
+// Some suggested trace levels.
+// You can use and define trace levels any way you want. Rlog comes with the
+// following definitions for a few basic trace levels, which you are free to
+// use if they fit your usage model.
+const (
+	TrPublic  TraceLevel = 1 // Message when entering a public, exported function
```

Comment:
this is always going to be used as rlog.TrPublic, etc., so instead of that we can have TraceLevelPublic, TraceLevelPrivate, TraceLevelFunction, etc.. Tr prefix is not preferred go way.